src/App.test.js

```diff
@@ -53,18 +54,30 @@ class MenubuttonFake {
   init() {}
 }
 
+const emptyResponse = { success: false, error: new Error('Dummy error') };
+
+const dopplerSitesClientDouble = {
+  getBannerData: async () => {
+    await timeout(0);
+    return emptyResponse;
+  },
+};
+
+const defaultDependencies = {
```

Comment:
What do you think about converting it into a function to avoid possible dirty dependencies between tests?

```suggestion
const createDefaultDependencies = () => ({
```

Reply:
We could also move this function to another place to allow us to reuse it easily.

Reply:
Yes maybe we can add in another pr and merge this to fix the test comments if you agree with that